scripts/upgrade_schema/update_schema_utility

```diff
@@ -0,0 +1,86 @@
+--
+-- Tables structures required by utility feature (uqc)
+--
+DROP TABLE IF EXISTS `uqc_outcome_hist`;
+DROP TABLE IF EXISTS `uqc_outcome_ent`;
+DROP TABLE IF EXISTS `uqc_outcome_dict`;
+--
+-- Table structure for table `uqc_outcome_dict`
+--
+CREATE TABLE `uqc_outcome_dict` (
+  `id_uqc_outcome` smallint(5) unsigned NOT NULL AUTO_INCREMENT,
+  `short_desc` varchar(50) NOT NULL,
+  `long_desc` varchar(150) DEFAULT NULL,
+  `iscurrent` tinyint(1) DEFAULT NULL COMMENT 'Catalog value still in use.',
+  `isvisible` tinyint(1) DEFAULT NULL COMMENT 'Is it visible in UI',
+  PRIMARY KEY (`id_uqc_outcome`),
+  UNIQUE KEY `unique_uqc_outcome_desc` (`short_desc`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1 COMMENT='Dictionary table for utility states';
+--
+-- Table structure for table `uqc_outcome_ent`
+--
+CREATE TABLE `uqc_outcome_ent` (
+  `id_uqc_outcome_ent` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+  `id_seq_composition` bigint(20) unsigned NOT NULL,
+  `id_uqc_outcome` smallint(5) unsigned NOT NULL COMMENT
+    'A foreign key referencing the id_seq_composition column of the seq_composition table',
+  `username` char(128) DEFAULT NULL COMMENT 'Web interface username',
+  `last_modified` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `modified_by` char(128) DEFAULT NULL COMMENT 'Last user to modify the row',
+  `last_reported` timestamp NULL DEFAULT NULL COMMENT 'When was reported the last time to LIMS',
+  `rationale` varchar(150) DEFAULT NULL COMMENT 'Explanation of utility qc choice',
+  PRIMARY KEY (`id_uqc_outcome_ent`),
+  UNIQUE KEY `id_seq_composition_UNIQUE` (`id_seq_composition`),
+  KEY `fk_uqc_outcome_ent_uqc_outcome_dict1` (`id_uqc_outcome`),
+  KEY `fk_uqc_outcome_ent_seq_composition` (`id_seq_composition`),
+  CONSTRAINT `fk_uqc_outcome_ent_uqc_outcome_dict1` FOREIGN KEY (`id_uqc_outcome`) REFERENCES uqc_outcome_dict (`id_uqc_outcome`) ON DELETE NO ACTION ON UPDATE NO ACTION,
+  CONSTRAINT `fk_uqc_outcome_ent_seq_composition` FOREIGN KEY (`id_seq_composition`) REFERENCES seq_composition (`id_seq_composition`) ON DELETE NO ACTION ON UPDATE NO ACTION
+) ENGINE=InnoDB DEFAULT CHARSET=latin1 COMMENT='Entity table for utility qc';
+--
+-- Table structure for table `uqc_outcome_hist`
+--
+CREATE TABLE `uqc_outcome_hist` (
+  `id_uqc_outcome_hist` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
When the ent table definition is changed, this table definition should be changed as well to match the ent.